cartography/intel/aws/resourcegroupstaggingapi.py

```diff
@@ -124,13 +125,18 @@ def get_short_id_from_lb2_arn(alb_arn: str) -> str:
     'iam:role': {'label': 'AWSRole', 'property': 'arn'},
     'iam:user': {'label': 'AWSUser', 'property': 'arn'},
     'kms:key': {'label': 'KMSKey', 'property': 'arn'},
+    'iam:group': {'label': 'AWSGroup', 'property': 'arn'},
+    'iam:role': {'label': 'AWSRole', 'property': 'arn'},
+    'iam:user': {'label': 'AWSUser', 'property': 'arn'},
```

Comment:
`iam:group`, `iam:role`, `iam:user` are duplicate keys

Reply:
removed it